gpii/node_modules/settingsHandlers/src/settingsHandlerUtilities.js

```diff
@@ -472,21 +476,21 @@ gpii.settingsHandlers.handleFileSolutionEntry = function (solutionEntry, modifie
  * is done by fluid.transform calls, and at the level of each solution
  * entry.
  */
-gpii.settingsHandlers.transformFilePayload = function (payload, modifier, parser, isWrite) {
+gpii.settingsHandlers.transformFilePayload = function (payload, modifier, parser, isWrite, fetcher) {
     return gpii.settingsHandlers.transformPayload(payload, function (solutionEntry) {
-        return gpii.settingsHandlers.handleFileSolutionEntry(solutionEntry, modifier, parser, isWrite);
+        return gpii.settingsHandlers.handleFileSolutionEntry(solutionEntry, modifier, parser, isWrite, fetcher);
     });
 };
 
-gpii.settingsHandlers.makeFileGet = function (parser) {
+gpii.settingsHandlers.makeFileGet = function (parser, fetcher) {
     return function (payload) {
-        return gpii.settingsHandlers.transformFilePayload(payload, gpii.settingsHandlers.getSettings, parser);
+        return gpii.settingsHandlers.transformFilePayload(payload, gpii.settingsHandlers.getSettings, parser, false, fetcher);
     };
 };
 
-gpii.settingsHandlers.makeFileSet = function (parser) {
+gpii.settingsHandlers.makeFileSet = function (parser, fetcher) {
```

Comment:
I would appreciate JSDocs for these functions.  Given that there are a whole bunch of functions that use the same parameters, seems like you'd just write once and copy/paste a few times.

Reply:
Added JSDocs